content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -45,18 +46,24 @@ including the `--oauth-no-pkce` flag when starting `chronograf`.
 - Flux improvements and additional functionality:
   - Add dashboard template variables to Flux query execution.
     Flux queries include a `v` record with a key value pair for each variable.
-  - Support dashboard template variables defined with Flux.
-  - Add Kapacitor Flux Tasks on the Manage Tasks page (read only).
+  - Support dashboard [template variables defined with Flux](/chronograf/v1.9/guides/dashboard-template-variables/#flux-query).
+  - Add [Kapacitor Flux tasks](/chronograf/v1.9/guides/advanced-kapacitor/#manage-kapacitor-flux-tasks)
+    on the Manage Tasks page (read only).
   - Provide documentation link when Flux is not enabled in InfluxDB 1.8+.
-  - Write to buckets when the Flux tab is selected.
+  - [Write to buckets using the Flux `to()` function](/chronograf/v1.9/guides/write-to-influxdb/#use-the-flux-to-function-in-a-query).
 - Filter fields in the Query Builder.
-- Select write precision when writing data.
-- Add PKCE to OAuth integrations.
-- Support GitHub Enterprise in the existing GitHub OAuth integration.
+- [Select write precision](/chronograf/v1.9/guides/write-to-influxdb/#upload-line-protocol-through-the-chronograf-ui)
+  when writing data through the Chronograf UI.
+- Support [GitHub Enterprise in the existing GitHub OAuth integration](/chronograf/v1.9/administration/managing-security/#configure-github-authentication).
 - Update the Queries page in the InfluxDB Admin section of the UI.
-- Set up InfluxDB Cloud and InfluxDB OSS 2.x connections with the `chronograf` CLI.
-- Add custom auto-refresh intervals.
-- Send multiple queries to dashboard.
+  - By default, sort queries by execution time in descending order.
+  - Sort queries by execution time or database.
```

Comment:
Would it be accurate to say: Add option to sort queries by....or does this sort happen automatically by some trigger?

Reply:
This happens if you click the column title.